src/Test.UnitTests.Sarif/Readers/DeferredCollectionsTests.cs

```diff
@@ -126,6 +156,34 @@ private static void CompareReadNormalToReadDeferred(string filePath)
             Assert.Equal(contextsCopy.Count, valueCount);
         }
 
+        private static void CompareReadNormalToReadDeferred(string filePath)
+        {
+            LogModelSampleBuilder.EnsureSamplesBuilt();
+            JsonSerializer serializer = new JsonSerializer();
```

Comment:
>JsonSerializer [](http://example.com/codeflow?start=12&length=14)

use 'var' here and elsewhere in the method

Reply:
done